cmd/hooks/wait-paths/wait_paths_test.go

```diff
@@ -0,0 +1,136 @@
+package main
+
+import (
+	"math"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+)
+
+func Test_Paths_EmptyString_NotAllowed(t *testing.T) {
+	args := []string{
+		"wait-paths",
+		"-p",
+		"",
+	}
+	app := newCliApp()
+	err := app.Run(args)
+	if err == nil {
+		t.Fatal("expected error, got nothing")
+	}
+	if !strings.Contains(err.Error(), "cannot be empty") {
```

Comment:
Can we predeclare the errors as `var errEmptyPaths = ...` and use `errors.Is` vs string comparison?
That way, you can also skip the initial `err == nil` check, since `errors.Is(nil, errEmptyPaths ) == false`.
(and similarly for the below tests?)